### ISSUE TITLE
Bookmark export/import column mismatch

### ISSUE THREAD

**Issue by MartinGebert** [anthonycr/Lightning-Browser#799]
Occurred on version 5.1.0.

When re-importing a bookmark export without any adaptions, the URL was used as title, and vice versa, resulting in unusable bookmark entries. Exchanging the column names "title" and "url" in the export file fixed this, the imported bookmarks were working correctly after that.

Note: Being able to import and export bookmarks (for backup and manual edit) is for me one of the important features to choose Lightning over Fennec. I would like to see some improvements in that area (e. g. being able to choose the export dir). Didn't try changing the order yet, but that's also important to me for keeping my bookmarks organized.

**Comment by chk1** [comment]
It is a known issue, unfortunately has been for a while, see #774

**Comment by Slion** [comment]
@MartinGebert This is fixed in [Fulguris](https://slions.net/resources/fulguris.10/).

**Comment by MartinGebert** [comment]
@Slion Nice hint, thanks. Tried it, bookmark import worked fine, but I wasn't able to get to my bookmarks by swipe from the left border as I'm now used to in Lightning. This, and some other details that leave the impression that fork is actually optimized for the F(x)tec, though it does some other things right. Another downside is that it's not available on F-Droid, and I would need to keep track of updates manually.

**Comment by ildar** [comment]
@Slion , minSdkVersion = 19
8-\
No, thanks.

**Comment by Slion** [comment]
> @Slion , minSdkVersion = 19 8-\ No, thanks.
@ildar What's wrong with that? Lightning is the same anyway. I was actually considering changing it to 20 something a couple of days ago.

**Comment by ildar** [comment]
Ah! (slapping my brow)
19 is perfectly fine and 20 also. 23 is good enough ;)
I just confused the numbering, thinking it's got Pie or something as
minimum ))
👍

**Comment by Slion** [comment]
> @Slion Nice hint, thanks. Tried it, bookmark import worked fine, but I wasn't able to get to my bookmarks by swipe from the left border as I'm now used to in Lightning. This, and some other details that leave the impression that fork is actually optimized for the F(x)tec, though it does some other things right. Another downside is that it's not available on F-Droid, and I would need to keep track of updates manually.

It is indeed developed primarely on F(x)tec Pro1, nevertheless most every Android device should be able to run it confortably all the same. Drawer swipes are disabled because they get in the way of full screen gesture navigation, think Android 10. Though I could add a setting option to enable them.

Feel free to share details of the thing you like or don't like. Just open an issue on the Fulguris project.

Fulguris is not currently published on F-Droid but you will get notified of updates availability when launching the app. You can also check for updates from Settings > About.

**Comment by Slion** [comment]
@ildar Yes, I think 23 was the one I was looking at but then I managed to get things done will remaining on19 anway...

**Comment by Slion** [comment]
@MartinGebert 

> I wasn't able to get to my bookmarks by swipe from the left border as I'm now used to in Lightning

This has been fixed in [Release v0.8.8](https://slions.net/resources/fulguris.10/updates#resource-update-64). Just make sure you unlock your drawers in the settings.
Settings > Look & Feel > Locked drawers
